src/main/java/com/verapdf/restapi/controller/JobController.java

```diff
@@ -0,0 +1,83 @@
+package com.verapdf.restapi.controller;
+
+import com.verapdf.restapi.dto.JobDTO;
+import com.verapdf.restapi.dto.JobFileDTO;
+import com.verapdf.restapi.dto.PathDTO;
+import com.verapdf.restapi.service.JobService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.servlet.mvc.method.annotation.MvcUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.Collections;
+import java.util.Map;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/jobs")
+public class JobController {
+
+    private JobService jobService;
+
+    @Autowired
+    public JobController(JobService jobService) {
+        this.jobService = jobService;
+    }
+
+    @PostMapping
+    public ResponseEntity<Map<String, String>> startJob() {
+        UUID uuid = jobService.createJob();
+        URI location = MvcUriComponentsBuilder
+                .fromMethodName(JobController.class, "getJob", uuid)
+                .build()
+                .toUri();
+        return ResponseEntity.created(location).body(Collections.singletonMap("jobId", uuid.toString()));
+    }
+
+    @GetMapping(value = "/{jobId}")
+    public ResponseEntity<JobDTO> getJob(@PathVariable UUID jobId) {
+        return ResponseEntity.ok(jobService.getJobDTO(jobId));
+    }
+
+    @DeleteMapping(value = "/{jobId}")
+    public ResponseEntity<String> closeJob(@PathVariable UUID jobId) {
+        jobService.closeJob(jobId);
+        return ResponseEntity.ok().build();
```

Comment:
change return type from Response entity to JobDTO

Reply:
when we are closing job, we need to return only ok, without JobDTO